openvoxserver/container-entrypoint.d/89-csr_attributes.sh

```diff
@@ -4,5 +4,5 @@ set -e
 
 if [ -n "${CSR_ATTRIBUTES}" ]; then
     echo "CSR Attributes: ${CSR_ATTRIBUTES}"
-    /opt/puppetlabs/puppet/bin/ruby /docker-entrypoint.d/89-csr_attributes.rb
+    /opt/puppetlabs/puppet/bin/ruby /container-entrypoint.d/89-csr_attributes.rb
```

Comment:
```suggestion
    my_directory="$(dirname "$0")"
    /opt/puppetlabs/puppet/bin/ruby "${my_directory}/89-csr_attributes.rb"
```

Same like https://github.com/OpenVoxProject/container-openvoxserver/pull/59/files#r2242548126

Reply:
Adjusted (in a slightly different manner).